persistence-modules/spring-data-elasticsearch/src/test/java/com/baeldung/elasticsearch/GeoQueriesManualTest.java

```diff
@@ -1,199 +1,192 @@
+import co.elastic.clients.json.JsonData;
+import co.elastic.clients.json.jackson.JacksonJsonpMapper;
+import co.elastic.clients.transport.ElasticsearchTransport;
+import co.elastic.clients.transport.rest_client.RestClientTransport;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
+import java.io.StringReader;
 import java.util.List;
-import java.util.stream.Collectors;
-
-import com.baeldung.spring.data.es.config.Config;
-
-import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
-import org.elasticsearch.action.admin.indices.refresh.RefreshRequest;
-import org.elasticsearch.action.index.IndexRequest;
-import org.elasticsearch.action.index.IndexResponse;
-import org.elasticsearch.action.search.SearchRequest;
-import org.elasticsearch.action.search.SearchResponse;
-import org.elasticsearch.client.RequestOptions;
-import org.elasticsearch.client.RestHighLevelClient;
-import org.elasticsearch.client.indices.CreateIndexRequest;
-import org.elasticsearch.common.geo.GeoPoint;
-import org.elasticsearch.common.geo.ShapeRelation;
-import org.elasticsearch.common.unit.DistanceUnit;
-import org.elasticsearch.index.query.GeoShapeQueryBuilder;
-import org.elasticsearch.index.query.QueryBuilder;
-import org.elasticsearch.index.query.QueryBuilders;
-import org.elasticsearch.search.SearchHit;
-import org.elasticsearch.search.builder.SearchSourceBuilder;
-import org.elasticsearch.xcontent.XContentType;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.locationtech.jts.geom.Coordinate;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 
 /**
  * This Manual test requires: Elasticsearch instance running on localhost:9200.
- * 
- * The following docker command can be used: docker run -d --name es762 -p
- * 9200:9200 -e "discovery.type=single-node" elasticsearch:7.6.2
+ * <p>
+ * The following docker command can be used:
+ * docker run -d --name elastic-test -p 9200:9200 -e "discovery.type=single-node" -e "xpack.security.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:8.9.0
  */
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(classes = Config.class)
+
+@Slf4j
 public class GeoQueriesManualTest {
 
     private static final String WONDERS_OF_WORLD = "wonders-of-world";
 
-    @Autowired
-    private RestHighLevelClient client;
+    private ElasticsearchClient client;
 
-    @Before
+    @BeforeEach
```

Comment:
Could we create and delete indices before and after all test instead of before and after each test?

Reply:
I just changed from JUnit 4 to JUnit 5. The behaviour of `@Before` in JUnit4 is the same as `@BeforeEach` in JUnit5. Creating and removing indexes gives us a clean environment to work within each unit test.

Reply:
Yes, @harry9656 they are equivalent. My point here was preferring reducing the cost of setting up the manual tests to gaining independence. Anyway, it's just a selection between the tradeoff, nothing super important here.